From 92998e522f158c718a096124dca06c7b9be7a0e3 Mon Sep 17 00:00:00 2001
From: "Tekriwal, Tanuj" <tanuj.tekriwal@intel.com>
Date: Tue, 28 Jan 2025 15:35:38 +0000
Subject: [PATCH] Update the health hal version dependency

This patch updates the health hal version dependency
from v3 to v4 based on latest code requirements.

Tests Done: Build proceed

Tracked-On: NA
Signed-off-by: Tekriwal, Tanuj <tanuj.tekriwal@intel.com>
---
 health/aidl/Android.bp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/health/aidl/Android.bp b/health/aidl/Android.bp
index ab13d03..7d66699 100644
--- a/health/aidl/Android.bp
+++ b/health/aidl/Android.bp
@@ -20,7 +20,7 @@ cc_defaults {
         "libcutils",
         "liblog",
         "libutils",
-        "android.hardware.health-V3-ndk",
+        "android.hardware.health-V4-ndk",
 
         // TODO(b/177269435): remove when BatteryMonitor works with AIDL HealthInfo.
         "libhidlbase",
@@ -39,7 +39,7 @@ cc_defaults {
     name: "libhealth_aidl_charger_defaults.intel",
     shared_libs: [
         // common
-        "android.hardware.health-V3-ndk",
+        "android.hardware.health-V4-ndk",
         "libbase",
         "libcutils",
         "liblog",
-- 
2.34.1

